src/manager/checkpoint.rs

```diff
@@ -33,16 +33,15 @@ use crate::lotus::LotusClient;
 const CHAIN_HEAD_REQUEST_PERIOD: Duration = Duration::from_secs(10);
 
 /// The `CheckpointSubsystem`. When run, it actively monitors subnets and submits checkpoints.
-struct CheckpointSubsystem {
+pub struct CheckpointSubsystem {
     /// The subsystem uses a `ReloadableConfig` to ensure that, at all, times, the subnets under
     /// management are those in the latest version of the config.
     config: ReloadableConfig,
 }
 
 impl CheckpointSubsystem {
     /// Creates a new `CheckpointSubsystem` with a configuration `config`.
-    #[allow(dead_code)]
-    fn new(config: ReloadableConfig) -> Self {
+    pub fn new(config: ReloadableConfig) -> Self {
```

Comment:
Here should have `Arc` for the `ReloadableConfig`?

Reply:
Yes!